### DISCUSSION TITLE
How to contribute translation of official documents on sqlalchemy's website?

### DISCUSSION THREAD

**Discussion by Winson-Huang** [sqlalchemy/sqlalchemy#7251] [Usage Questions] [answered]
Is there a multi-language plan of sqlalchemy's official documents? I think the documents are the best learning resources for beginners, and for sure native language is more easy to understand even to "ESL" people, so I wonder if there is a way to contribute translation on official website?

**Comment by CaselIT** [comment, marked as answer]
Hi,

SQLAlchemy does not currently have an official documentation translation story. There are some 3rd party translations, like this one https://github.com/sqlalchemy/sqlalchemy/issues/4578#issuecomment-477882029, but I cannot vouch for their quality.

The main issue with translations is that they require a lot of effort to both produce them and then to keep them up to date, and that's something that the SQLAlchemy team cannot at the moment provide. We are open on supporting 3rd parties that would like to provide a better story regarding translations.

One thing we could investigate is to maybe integrate one of the automated translation services in the documentation website, like google translate. They will not be 100% accurate but they do  solve the maintenance burden.
Looking at the native Italian translation it seems passable: https://docs-sqlalchemy-org.translate.goog/en/14/?_x_tr_sl=auto&_x_tr_tl=it&_x_tr_hl=en-GB&_x_tr_pto=nui. I fear I cannot judge google Chinese quality though https://docs-sqlalchemy-org.translate.goog/en/14/?_x_tr_sl=auto&_x_tr_tl=zh-CN&_x_tr_hl=en-GB&_x_tr_pto=nui
